homeassistant/components/iammeter/sensor.py

```diff
@@ -40,6 +58,51 @@
 PLATFORM_TIMEOUT = 8
 
 
+def _migrate_to_new_unique_id(
+    hass: HomeAssistant, model=DEVICE_3080, serial_number: str = ""
```

Comment:
This function is never called without all parameters, please remove the default values.
```suggestion
    hass: HomeAssistant, model:str, serial_number: str
```